stdlib/public/core/Bool.swift

```diff
@@ -170,6 +170,13 @@ extension Bool : _ExpressibleByBuiltinBooleanLiteral, ExpressibleByBooleanLitera
   }
 }
 
+// This is an old intrinsic known by the compiler, but exists solely for ABI
+// compatibility at this point.
+@usableFromInline @_transparent
```

Comment:
Can you remove the @_transparent as well?

@airspeedswift Do you know of an example where a user binary was calling this, or is it purely defensive?

Reply:
Off the top of my head, I know the compiler would call this for `IsExpr`s.
```swift
let x = 3 is Int
```

Reply:
Right, I would expect any call emitted by the compiler would directly apply the function, so it would be inlined away. A user would have to explicitly write a call to _getBool() and make use of the result, which is non-trivial since it is a built-in type. But perhaps we had a bug where it wasn't inlined away somewhere?

Reply:
Purely defensive. Like you say, if there's no way for a user to call it. If you're certain there's no way the compiler could emit this symbol without just inlining the implementation, we can leave it removed, but it'd be a shame if we're wrong so the reasoning needs to be solid. WDYT?

Reply:
Could `_getBool(_:)` be moved to the [LegacyABI.swift](https://github.com/apple/swift/blob/master/stdlib/public/core/LegacyABI.swift) file?

And possibly use one of the following:

- `@available(swift, obsoleted: 5.0)`
- `@available(*, unavailable, renamed: "Bool.init(_:)")`
- `@available(*, unavailable, renamed: "Bool.init(_builtinBooleanLiteral:)")`